compiler/tools/fpp-to-cpp/test/component/ActiveStateMachines_S1.cpp

```diff
@@ -0,0 +1,103 @@
+
+// ======================================================================
+// \title  ActiveStateMachines_S1.cpp
+// \author Auto-generated
+// \brief  cpp file for state machine ActiveStateMachines_S1
+//
+// ======================================================================            
+    
+#include "stdio.h"
+#include "assert.h"
+#include "Fw/Types/SMSignalsSerializableAc.hpp"
```

Comment:
We need to remove this header include. `SMSignals` is no longer used.

Reply:
I removed this.

Reply:
I fixed this. I also removed the dependencies on stdio and assert. These are not allowed in F Prime flight code. @garthwatney Please update the STARS code generator to match.